Packs/Cymulate/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cymulate",
     "description": "Multi-Vector Cyber Attack, Breach and Attack Simulation",
```

Comment:
Does not really explain why this attack simulations are important. How about we change it like this?
-> "You can now verify your security posture on-demand using the Cymulate integration, which allows you to launch simulations of cyber attacks, breach, and attacks against yourself."

Reply:
This description is taken from the Cymulate's first integration and was approved by them again when we started working on the second integration version.